cypress/support/service/saleschannel/fixture/order.fixture.js

```diff
@@ -7,22 +7,24 @@ class OrderFixtureService extends SalesChannelFixtureService {
                 this.apiClient.setAccessKey(result);
             })
             .then(() => {
-                return this.apiClient.post(`/${Cypress.env('apiVersion')}/customer/login`, JSON.stringify({
+                return this.apiClient.post(`/${Cypress.env('apiVersion')}/account/login`, JSON.stringify({
                     username: customer.username,
                     password: customer.password
                 }));
             })
-            .then((contextToken) => {
-                return this.apiClient.setContextToken(contextToken['sw-context-token']);
+            .then((response) => {
+                return this.apiClient.setContextToken(response.data['contextToken']);
```

Comment:
`response.data.contextToken` - you don't need the array notation here. This is only necessary when the key contains characters like hyphens